### PR TITLE
Wire up toolbar functionality

### PR DESCRIPTION
**Pull request by apalaniuk**

### CODE REVIEW

File: d2l-pdf-viewer-toolbar.html [hunk 1]
```diff
@@ -191,8 +189,8 @@
 					class="toolbarField pageNumber"
 					type$="[[_pageNumberInputType(hasPageLabels)]]"
 					max$="[[_pageNumberInputMax(pagesCount)]]"
-					value$="[[_pageNumberInputValue(hasPageLabels, pageNumber, pageLabel)]]"
-					on-change="_onChangePageNumber">
+					value="{{_inputPageNumber::input}}"
```
Comment (apalaniuk) [inline d2l-pdf-viewer-toolbar.html]: Seems necessary if we're using binding in order for the value to be reset when invalid. Could also not bind it I suppose - this approach precludes using page labels for now (or the type just needs to be a string and coerced a little differently), but page labels have some quirks in the consuming element so aren't going to be immediately available anyway, so will just be page number-focused for the moment.

File: d2l-pdf-viewer-toolbar.html [hunk 2]
```diff
@@ -261,77 +273,58 @@
 				return pageNumber >= pagesCount;
 			},
 			// compute zoom out button disabled
-			_zoomOutButtonDisabled: function(pageScale) {
-				return pageScale <= this._minScale;
+			_zoomOutButtonDisabled: function(pageScale, minPageScale) {
+				return pageScale <= minPageScale || Math.abs(pageScale - minPageScale) < Number.EPSILON;
```
Comment (ChrisKraljevicD2L) [inline d2l-pdf-viewer-toolbar.html]: Where does `Number.EPSILON` come from?

File: d2l-pdf-viewer.html [hunk 3]
```diff
@@ -18,13 +18,14 @@
 				background-color: grey;
 			}
 
-			div#viewerContainer {
-				height: 100%;
+			#viewerContainer {
+				position: relative;
```
Comment (apalaniuk) [inline d2l-pdf-viewer.html]: Required for scroll-into-view to work.

File: d2l-pdf-viewer-toolbar.html [hunk 4]
```diff
@@ -261,77 +273,58 @@
...
+				return pageScale >= maxPageScale || Math.abs(pageScale - maxPageScale) < Number.EPSILON;
 			},
 			// --events--
 			// click previous page
 			_onTapPreviousPage: function() {
 				this.dispatchEvent(
-					new CustomEvent(
-						'd2l-pdf-viewer-toolbar-previous',
-						{
-							bubbles: true,
```
Comment (apalaniuk) [inline d2l-pdf-viewer-toolbar.html]: I definitely said to add these earlier, but unnecessary in how events are wired up

File: d2l-pdf-viewer.html [hunk 5]
```diff
@@ -68,57 +73,67 @@
...
 				}
 			},
 			ready: function() {
-				var container = this.$.viewerContainer;
-
 				// (Optionally) enable hyperlinks within PDF files.
 				var pdfLinkService = new pdfjsViewer.PDFLinkService();
 
-				// (Optionally) enable find controller.
-				var pdfFindController = new pdfjsViewer.PDFFindController({
```
Comment (apalaniuk) [inline d2l-pdf-viewer.html]: Removing some unused things/comments for now for clarity on what's being used. Trivial to re-add when we explicitly add this functionality

File: d2l-pdf-viewer-toolbar.html (hunk 2 above)
Comment (apalaniuk) [reply d2l-pdf-viewer-toolbar.html]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON

**Pull request merged**